cmd/ingress-operator/render.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
+)
+
+func NewRenderCommand() *cobra.Command {
+	var options struct {
+		OutputDir string
+		Prefix    string
+	}
+
+	var command = &cobra.Command{
+		Use:   "render",
+		Short: "Render base manifests",
+		Long:  `render emits the base manifest files necessary to support the creation of an ingresscontroller resource.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := render(options.OutputDir, options.Prefix); err != nil {
+				log.Error(err, "error rendering")
+				os.Exit(1)
+			}
+		},
+	}
+
+	command.Flags().StringVarP(&options.OutputDir, "output-dir", "o", "-", "manifest output directory. Use '-' for stdout.")
+	command.Flags().StringVarP(&options.Prefix, "prefix", "p", "", "optional prefix for rendered filenames.")
+
+	return command
+}
+
+func render(dir string, prefix string) error {
+	files := []string{
+		manifests.CustomResourceDefinitionManifest,
+		manifests.NamespaceManifest,
+	}
+
+	if dir == "-" {
+		for _, file := range files {
+			fmt.Println("---")
+			fmt.Printf("# file: %s\n", filepath.Base(file))
+			fmt.Println(manifests.MustAssetString(file))
+			fmt.Println("...")
```

Comment:
I tried the render command and got the following output (showing the first few lines):
```yaml
---
# file: 00-custom-resource-definition.yaml

---
apiVersion: apiextensions.k8s.io/v1beta1
kind: CustomResourceDefinition
metadata:
  creationTimestamp: null
  name: ingresscontrollers.operator.openshift.io
```
On investigation, I realized `manifests/00-custom-resource-definition.yaml` already has a `---` marker.  We need to delete either the print statements for the markers or the `---` marker in the file.

Reply:
Of course if this loop does not emit markers, that means we also need to add a `---` marker to `manifests/00-namespace.yaml` so that they will be separated when the render command prints them.  I think best would be to remove the marker from `manifests/00-custom-resource-definition.yaml` and keep the print statements as they are.

Reply:
We've spent way more time on this particular code branch than is reasonable — it's not actually used for anything but debugging. I went ahead and deleted the entire functionality. Can this now merge?